WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -4488,78 +4517,92 @@ templates.class.DEMONHUNTER = {
+        { spell = 389847, type = "buff", unit = "player", talent = 112850 }, -- Felfire Haste
         { spell = 389890, type = "buff", unit = "player", talent = 112944 }, -- Tactical Retreat
-        { spell = 390145, type = "buff", unit = "player", talent = 112929 }, -- Inner Demon
+        { spell = 390145, type = "buff", unit = "player", talent = 112947 }, -- Inner Demon
+        { spell = 390192, type = "buff", unit = "player", talent = 112827 }, -- Ragefire
         { spell = 390195, type = "buff", unit = "player", talent = 112958 }, -- Chaos Theory
+        { spell = 390212, type = "buff", unit = "player", talent = 117765 }, -- Restless Hunter
         { spell = 391215, type = "buff", unit = "player", talent = 112950 }, -- Initiative
         { spell = 427641, type = "buff", unit = "player", talent = 117744 }, -- Inertia
-
+        { spell = 427901, type = "buff", unit = "player", talent = 115246 }, -- Deflecting Dance
+        { spell = 442435, type = "buff", unit = "player" }, -- Glaive Flurry
+        { spell = 442442, type = "buff", unit = "player" }, -- Rending Strike
+        { spell = 442503, type = "buff", unit = "player", herotalent = 117503 }, -- Warblade's Hunger
+        { spell = 442688, type = "buff", unit = "player", herotalent = 117516 }, -- Thrill of the Fight
+        { spell = 442715, type = "buff", unit = "player" }, -- Blade Ward
+        { spell = 442788, type = "buff", unit = "player", herotalent = 123046 }, -- Incorruptible Spirit
+        { spell = 444661, type = "buff", unit = "player", herotalent = 117512 }, -- Art of the Glaive
+        { spell = 444929, type = "buff", unit = "player", herotalent = 117508 }, -- Evasive Action
+        { spell = 452416, type = "buff", unit = "player", herotalent = 117514 }, -- Demonsurge
+        { spell = 452550, type = "buff", unit = "player", herotalent = 117506 }, -- Monster Rising
+        { spell = 453239, type = "buff", unit = "player", herotalent = 117499 }, -- Student of Suffering
+        { spell = 453314, type = "buff", unit = "player", herotalent = 117513 }, -- Enduring Torment
       },
       icon = 1247266
     },
     [2] = {
       title = L["Debuffs"],
       args = {
         { spell = 1490, type = "debuff", unit = "target" }, -- Chaos Brand
-        { spell = 179057, type = "debuff", unit = "target", talent = 112911 }, -- Chaos Nova
-        { spell = 185245, type = "debuff", unit = "target" }, -- Torment
-        { spell = 198813, type = "debuff", unit = "target", talent = 112853 }, -- Vengeful Retreat
-        { spell = 200166, type = "debuff", unit = "target" }, -- Metamorphosis
         { spell = 204598, type = "debuff", unit = "target" }, -- Sigil of Flame
-        { spell = 207685, type = "debuff", unit = "target", talent = 112859 }, -- Sigil of Misery
-        { spell = 211881, type = "debuff", unit = "target" }, -- Fel Eruption
-        { spell = 213405, type = "debuff", unit = "target", talent = 112912 }, -- Master of the Glaive
```

Comment:
Havoc still seems to have Chaos Nova, Sigil of Miser, Vengeful Retreat debuff, and Torment and Fel Eruption